std/experimental/allocator/building_blocks/bucketizer.d

```diff
@@ -271,3 +272,32 @@ struct Bucketizer(Allocator, size_t min, size_t max, size_t step)
     assert(a.reallocate(b, 0));
     assert(b is null);
 }
+
+@system unittest
+{
+    import std.experimental.allocator.building_blocks.bitmapped_block : BitmappedBlock;
+    import std.experimental.allocator.gc_allocator : GCAllocator;
+
+    Bucketizer!(BitmappedBlock!(64, 8, GCAllocator), 1, 512, 64) a;
+    for (auto i = 0; i < a.buckets.length; ++i) {
```

Comment:
Braces on their own line, and this can also probably be written as a `foreach` loop.